e2e/process-services/widgets/people_widget.e2e.ts

```diff
@@ -0,0 +1,102 @@
+import { TasksPage } from '../../pages/adf/process_services/tasksPage';
+import { Widget } from '../../pages/adf/process_services/widgets/widget';
+import CONSTANTS = require('../../util/constants');
+import TestConfig = require('../../test.config');
+import resources = require('../../util/resources');
+
+describe('People widget', () => {
+
+    let loginPage = new LoginPage();
+    let processUserModel;
+    let taskPage = new TasksPage();
+    let widget = new Widget();
+    let alfrescoJsApi;
+    let appsActions = new AppsActions();
+    let appModel;
+    let app = resources.Files.WIDGET_CHECK_APP.ADD_PEOPLE;
+    let deployedApp, process;
+
+    beforeAll(async (done) => {
+        let users = new UsersActions();
+
+        alfrescoJsApi = new AlfrescoApi({
+            provider: 'BPM',
+            hostBpm: TestConfig.adf.url
+        });
+
+        await alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+
+        processUserModel = await users.createTenantAndUser(alfrescoJsApi);
+
+        await alfrescoJsApi.login(processUserModel.email, processUserModel.password);
+        appModel = await appsActions.importPublishDeployApp(alfrescoJsApi, resources.Files.WIDGET_CHECK_APP.file_location);
+
+        let appDefinitions = await alfrescoJsApi.activiti.appsApi.getAppDefinitions();
+        deployedApp = appDefinitions.data.find((currentApp) => {
+            return currentApp.modelId === appModel.id;
+        });
+        process = await appsActions.startProcess(alfrescoJsApi, appModel, app.processName);
+        loginPage.loginToProcessServicesUsingUserModel(processUserModel);
+        done();
+    });
+
+    beforeEach(() => {
+        let urlToNavigateTo = `${TestConfig.adf.url}/activiti/apps/${deployedApp.id}/tasks/`;
+        browser.get(urlToNavigateTo);
+        taskPage.filtersPage().goToFilter(CONSTANTS.TASK_FILTERS.MY_TASKS);
+        taskPage.formFields().checkFormIsDisplayed();
+    });
+
+    afterAll(async (done) => {
+        await alfrescoJsApi.activiti.processApi.deleteProcessInstance(process.id);
+        await alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+        await alfrescoJsApi.activiti.adminTenantsApi.deleteTenant(processUserModel.tenantId);
+        done();
+    });
+
+    it('[C212870] should check People widget', () => {
+        taskPage.formFields().checkWidgetIsHidden(app.FIELD.widget_id);
+        widget.checkboxWidget().clickCheckboxInput(app.FIELD.checkbox_id);
+        taskPage.formFields().checkWidgetIsVisible(app.FIELD.widget_id);
+
+        let admin = processUserModel.firstName + ' ' + processUserModel.lastName;
+        widget.peopleWidget().insertUser(app.FIELD.widget_id, admin.charAt(0));
+        widget.peopleWidget().checkDropDownListIsDisplayed();
+        widget.peopleWidget().checkUserIsListed(admin);
+        widget.peopleWidget().selectUserFromDropDown(admin);
+    });
+
+    it('[C274707] Add people widget - Visibility', () => {
```

Comment:
Could you change the name to : Should be possible to set visibility properties for People Widget ?